kafkamgmt/apiv1/errors/error_sdk.go

```diff
@@ -6,9 +6,6 @@ import (
 	kafkamgmtclient "github.com/redhat-developer/app-services-sdk-go/kafkamgmt/apiv1/client"
```

Comment:
package name needs to be changed to `errors`.

Reply:
actually we getting conflict due to "errors" being the core package name.  I would rename it to apierror to avoid that

Reply:
Actually no - the error is caused by having two different package names within the same directory. `errors` should work, but at the same time it might be best to rename it to avoid conflicts.